cmd/helm/testdata/testcharts/chart-with-subchart-notes/requirements.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - name: subchart-with-notes
+    version: 0.0.1
```

Comment:
Can add to parent `Chart.yaml` file instead and remove this file (new Helm 3 format).

Reply:
Good call.  Just pushed.